release_notes/ocp-4-15-release-notes.adoc

```diff
@@ -590,6 +592,14 @@ If you are receiving pod security violations, see the following resources:
 
 * If necessary, you can set a custom admission profile on the namespace or pod by setting the `pod-security.kubernetes.io/enforce` label.
 
+[discrete]
+[id="ocp-4-15-auth-generated-secrets"]
+==== Secrets are no longer automatically generated when the integrated {product-registry} is disabled
+
+If you disable the `ImageRegistry` capability or if you disable the integrated {product-registry} in the Cluster Image Registry Operator's configuration, a service account token secret and image pull secret are no longer generated for each service account.
+
+For more information, see xref:../nodes/pods/nodes-pods-secrets.adoc#auto-generated-sa-token-secrets_nodes-pods-secrets[Automatically generated secrets].
```

Comment:
Note to reviewers: the link text doesn't match the page currently, but it will match once #71256 is merged.